frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -147,24 +148,28 @@ export const PaymentInput = (): JSX.Element => {
           )
         },
         validateMin: (val) => {
+          if (minPaymentAmountCents === undefined) return true
           return (
-            Number(val?.trim()) >= minPaymentAmount ||
+            // val is in dollars
+            (val && dollarsToCents(val) >= minPaymentAmountCents) ||
             `Please enter a payment amount above ${formatCurrency(
-              minPaymentAmount,
+              Number(centsToDollars(minPaymentAmountCents)),
```

Comment:
nit: is `formatCurrency(Number(...))` still necessary? 🤔 line 165 too (if no, can remove `formatCurrency` function altogether I think

Reply:
Yes, so that the currency is correctly displayed. Otherwise we could just put `$${centsToDollars(...)}` with `$` manually but using the currency format seems more correct.